content/en/blog/_posts/2022-09-14-pod-has-network-condition.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
+date: 2022-09-14
+slug: pod-has-network-condition
+---
+
+**Author:**
+Deep Debroy (Apple)
+
+Kubernetes 1.25 introduces Alpha support for a new kubelet-managed pod condition
+in the status field of a pod: `PodHasNetwork`. The kubelet will use the
+`PodHasNetwork` condition to accurately surface the initialization state of a
+pod from the perspective of pod sandbox creation and network configuration by a
+CRI plugin in coordination with CNI plugins. The kubelet starts to pull
+container images and start individual containers (including init containers)
+after the status of the PodHasNetwork condition is set to `True`. Metrics
+collection services that report latency of pod initialization from a cluster
+infrastructural perspective (i.e. agnostic of per container characteristics like
+image size or payload) can utilize the `PodHasNetwork` condition to accurately
+generate SLIs. Certain operators or controllers that manage underlying pods may
+utilize the PodHasNetwork condition to optimize the set of actions performed
```

Comment:
Please enclose all `PodHasNetwork` with single quotes across of this whole docs, as in above cases.

Reply:
Done